### ISSUE TITLE
Updating tree frequently

### ISSUE THREAD

**Issue by palashjhabak** [SubZane/flyPanels#6]
I have a scenario in which the tree is updates very frequently. For newer elements to have events attached to them what, according to you is the best way?

I tried destroy and reconstruct the plugin but that seems to be very slow (the expanding function stops for a few soconds) somehow

**Comment by SubZane** [comment]
That's true. Destroying and recreating flypanels in order to just update the tree is not a good option.

Currently there is no methods avilable to just simply re-create the tree, but it sounds like a useful feature. I'll add it to the next version.
